web/ui/react-app/src/pages/targets/EndpointLink.tsx

```diff
@@ -8,23 +8,22 @@ export interface EndpointLinkProps {
 
 const EndpointLink: FC<EndpointLinkProps> = ({ endpoint, globalUrl }) => {
   let url: URL;
+  let search = '';
+  let invalidURL = false;
   try {
     url = new URL(endpoint);
   } catch (err: unknown) {
-    const error = err as Error;
-    return (
-      <Alert color="danger">
-        <strong>Error:</strong> {error.message}
-      </Alert>
-    );
+    invalidURL = true;
+    if (endpoint.indexOf('?') > -1) search = endpoint.substring(endpoint.indexOf('?'));
+    url = new URL('http://0.0.0.0' + search);
```

Comment:
Would be good to add a comment somewhere in this branch to explain that the URL might actually have been valid, but that browsers don't support that IPv6 edge case.

Reply:
comment added in last commit